src/containers/Transactions/simpleTab.scss

```diff
@@ -31,20 +32,24 @@
     .row {
       display: flex;
       overflow: hidden;
-      padding: 20px 10px;
+      padding: 0px 15px;
+      padding-left: 0px;
```

Comment:
FYI it's considered good CSS style to not include a unit for 0 values

Reply:
I disagree because it could be undoing something.  What is generally good practice is `0` vs `0px`.